kibana.d.ts

```diff
@@ -40,6 +41,13 @@ export namespace Legacy {
   export type SavedObjectsService = LegacyKibanaServer.SavedObjectsService;
   export type Server = LegacyKibanaServer.Server;
 
+  export type InitPluginFunction = (server: Server) => void;
+  export interface UiExports {
+    injectDefaultVars: (server: Server) => { [key: string]: any };
+  }
```

Comment:
Can `InitPluginFunction` and `UiExports` be re-exported from `LegacyKibanaPluginSpec`, since they're already included once there?

Reply:
yes that makes much more sense.